Update Plan Package Version

lgiancol opened the pull request:
Updated pkg_version 2.7.1 -> 2.7.4 in support of 2021 Q2 core plans refresh.

lgiancol committed:
Bumped version 2.7.1 -> 2.7.2

smandaka committed:
Bumped 2.7.3 version

sajjaphani committed:
updated the bundler help output match

Signed-off-by: Phani Sajja <psajja@progress.com>

sajjaphani committed:
bundler help is different in local and azure

Signed-off-by: Phani Sajja <psajja@progress.com>

mwrock commented on controls/ruby27_works.rb:
```diff
@@ -8,20 +8,21 @@
   full_suite = {
     "bundle" => {
-      command_output_pattern: /Bundler\s+commands:/,
+      # Update the command output match for v2.7.3
```
I don't think this comment is necessary. Future developers likely will not wonder why a line of code changed from a previous version. Its better for a commit message.

sirajrauff committed:
Updating to 2.7.4

From https://github.com/habitat-sh/core-plans/pull/3995/commits/74d88f6cbbe882f42b3e765119cabc5ff7bcacb7
Courtesy of @tas50

Signed-off-by: Siraj Rauff <sirajrauff@gmail.com>
